Codecademy/gamut
# refactor(GlobalFooter): Copy and Layout Updates

### Overview

- Restructure the footer layout, per design spec. 
- Add additional subjects and languages to the footer, as specified in the design. 
- Maintain existing analytic context and target values. For new links, use the structure
                      **context = ‘global_footer’
                      target = ‘[link_title]’**

:white_check_mark: Approved by Mark and Tamar 





### PR Checklist

- [x] Related to designs: https://www.figma.com/file/qwSSkbOBqw9m2FbENE1tFV/Footer-Update?node-id=79%3A524&t=RRA1AZxfoVZiv5r6-0
- [x] Related to JIRA ticket: [DISC-1013]

#### Testing Instructions



<ins>Design references</ins> 
[Desktop](https://www.figma.com/file/qwSSkbOBqw9m2FbENE1tFV/Footer-Update?node-id=95%3A2488&t=l6qeZ3w53N6nQp0L-0) 
[Tablet
](https://www.figma.com/file/qwSSkbOBqw9m2FbENE1tFV/Footer-Update?t=LYkeMCr54laTsMku-0)[Mobile](https://www.figma.com/file/qwSSkbOBqw9m2FbENE1tFV/Footer-Update?node-id=130%3A1369&t=LYkeMCr54laTsMku-0)

1. Open [storybook GlobalHeader](https://63c83a4b7ad99d105c204bce--gamut-preview.netlify.app/?path=/docs/brand-labs-organisms-globalfooter--global-footer) and see new GlobalHeader and also check out the Hide Pricing section to make sure Plans doesn't show. Note that you should use the PR env to test different screen sizes as the new footer in storybook is squished a bit. 
2. See new Footer live in the PR envs and check different view ports.

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/34349/ 'Named link title')  | [Monolith Env](https://pr-34349-monolith.dev-eks.codecademy.com/ 'Named link title') |
| Monorepo       | [Monorepo Link](https://github.com/codecademy-engineering/mono/pull/1084/ 'Named link title')  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-1084/ 'Named link title')   |



[DISC-1013]: https://codecademy.atlassian.net/browse/DISC-1013?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
